translate/storage/mozilla_lang.py

```diff
@@ -116,6 +130,13 @@ def parse(self, lines):
     def serialize(self, out):
         if self.is_active or self.mark_active:
             out.write(b"## active ##\n")
+        for header in self._headers:
+            out.write(bytes("%s\n" % header))
+        if self._headers:
+            out.write(b"\n\n")
         for unit in self.units:
             out.write(six.text_type(unit).encode('utf-8'))
             out.write(b"\n\n\n")
+
+    def getlangheaders(self):
```

Comment:
i want to explicitly make this the public method - afaict the only other format that has headers implemented is po - and in that case its called getpoheaders (iirc) - because they are format-specific